elixir/phoenix_live_view_dashboard.md

```diff
@@ -0,0 +1,325 @@
+mix archive.install hex mix_templates
+mix archive.install hex mix_generator
+
+// install my template
+mix template.install hex gen_template_phx_live_view
+
+// create your project
+mix gen phx_live_view MyAwesomeLiveViewProject
+```
+
+For my dashboard I wanted to have some boilerplate styling and libraries to make my design work a little easier (because that is just not my cup-of-thea). So I created an additional template that sets up a few other things. Namely bootstrap and sass and some initial dashboard layout.
+
+You can ofcourse use this template as a starting just as easily as the basic one:
+
+```
+//Install the template
+mix template.install hex gen_template_phx_live_view_dashboard
+
+// create your project
+mix gen phx_live_view_dashboard MyAwesomeLiveViewDashboard
+```
+
+If you start up this either of these templates you'll see that it already has a LiveView clock running. 
+
+### Countdown To ElixirConf
+
+With the upcoming ElixirConfEU in Prague, let's see if we can get some more realtime data on the screen that relates to this upcoming event. Let's start by creating a countdown timer to the start of the conference!
+
+Adding a new LiveView component can be done in two simple steps.
+
+1. create a LiveView in `lib/yourproject_web/live/LiveViewModule.ex`
+This LiveView must implement two functions, `mount/2` and `render/1`. 
+
+2. Render the view on a page.
+This is simple using the `live_render/2` function.
+Here is the exmple from the clock that is in the template.
+```elixir
+  <%= live_render(@conn, ElixirConfLiveViewWeb.Clock) %>
+```
+
+Let's create the countdown timer. I'll use the awesome Timex library to make my life a little easier. :) 
+
+So let's add that to `mix.exs`
+```elixir
+...
+  {:timex, "~> 3.1"},
+...
+```
+
+And here is the ViewModule implementation of the countdown timer. Once the livesocket is connected it fires off a timer that send a message to the process every second to get the current duration to the start of ElixirConf. That triggers and update to the socket.assigns, which in it's turn triggers the render function to rerender the component on the page. Sounds too easy? Well it is too easy. Great stuff!.
+
+```elixir
+defmodule ElixirConfLiveViewWeb.CountdownToElixirconf do
+  use Phoenix.LiveView
+  use Phoenix.HTML
+  alias Timex.{ Interval, Duration, Format.Duration.Formatter }
+  
+  def render(assigns) do
+    ~L"""
+    <div class="countdown-to-elixirconf">
+      <h3><%= @time_to_conf %></h3>
+    </div>
+    """
+  end
+
+  def mount(_session, socket) do
+    if connected?(socket), do: :timer.send_interval(1000, self(), :tick)
+
+    {:ok, put_time_to_conf(socket)}
+  end
+
+  def handle_info(:tick, socket) do
+    {:noreply, put_time_to_conf(socket)}
+  end
+
+  defp put_time_to_conf(socket) do
+    assign(socket, time_to_conf: time_to_conf())
+  end
+
+  defp time_to_conf do
+    cond do
+      time_to_elixirconf() > Timex.now ->
+        "It's Alive!! Alive!"
+      true ->
```

Comment:
you may as well do `_` to signal the value doesn't matter, as this is the catch all case. right? I'd argue to just use if/else even although I'm not sure what's preferred in Elixir

Reply:
the `_` is indeed used for case statements default clause
`true` is needed here, because it is a `cond`